### DISCUSSION TITLE
Quit App Button

### DISCUSSION THREAD

**Discussion by Kowersky** [zauberzeug/nicegui#3532] [Q&A]
### Question

Hi. I am making my application and to close the application I am using 
```py
async def quitApp():
    ui.notify('I'm shutting down', type='ongoing', timeout='1')
    await asyncio.sleep(1)
    app.shutdown()
```

When I click the button from shutdown 2 processes stay. 

Video: https://i.imgur.com/PWmhvZi.mp4

**Comment by falkoschindler** [comment]
Hi @Kowersky,

Please provide a minimal reproducible code example.
[Why is it important to provide a minimal reproducible example?](https://github.com/zauberzeug/nicegui/wiki/FAQs#why-is-it-important-to-provide-a-minimal-reproducible-example)
And it seems like you built and packaged a native app, is that right? Does the problem also occur with an unpackaged app running in the browser?

**Reply by Kowersky** [reply]
> Hi @Kowersky,
> 
> Please provide a minimal reproducible code example. [Why is it important to provide a minimal reproducible example?](https://github.com/zauberzeug/nicegui/wiki/FAQs#why-is-it-important-to-provide-a-minimal-reproducible-example) And it seems like you built and packaged a native app, is that right? Does the problem also occur with an unpackaged app running in the browser?

I have processes only when its packaged by nuitka

**Reply by Kowersky** [reply]
> > Hi @Kowersky,
> > Please provide a minimal reproducible code example. [Why is it important to provide a minimal reproducible example?](https://github.com/zauberzeug/nicegui/wiki/FAQs#why-is-it-important-to-provide-a-minimal-reproducible-example) And it seems like you built and packaged a native app, is that right? Does the problem also occur with an unpackaged app running in the browser?
> 
> I have processes only when its packaged by nuitka

without converting works fine
